.changes/1071-chat-screen-improvements.md

```diff
@@ -0,0 +1,2 @@
+- Hide emoji option after sent
```

Comment:
Please add the mentioning of where the change is occurred, in this case its the room message actions of the chat feature. These logs should be simply written with clear detail and are not for the developers but for users.  Also specify whether its a fix or an addition.

Reply:
Sure @gtalha07. I will update details accordingly. Thanks for your feedback.